src/core/Akka.Remote/Serialization/DaemonMsgCreateSerializer.cs

```diff
@@ -81,55 +82,61 @@ public override byte[] ToBinary(object obj)
         {
             var msg = obj as DaemonMsgCreate;
             if (msg == null)
-                throw new ArgumentException("Can't serialize a non-DaemonMsgCreate message using DaemonMsgCreateSerializer");
-       
-            DaemonMsgCreateData daemonBuilder = DaemonMsgCreateData.CreateBuilder()
-                .SetProps(GetPropsData(msg.Props))
-                .SetDeploy(GetDeployData(msg.Deploy))
-                .SetPath(msg.Path)
-                .SetSupervisor(SerializeActorRef(msg.Supervisor))
-                .Build();
-
-            return daemonBuilder.ToByteArray();
+            {
+                throw new ArgumentException(
+                    "Can't serialize a non-DaemonMsgCreate message using DaemonMsgCreateSerializer");
+            }
+
+            var daemon = new DaemonMsgCreateData
+            {
+                Props = GetPropsData(msg.Props),
+                Deploy = GetDeployData(msg.Deploy),
+                Path = msg.Path,
+                Supervisor = SerializeActorRef(msg.Supervisor)
+            };
+
+            return daemon.ToByteArray();
         }
 
         private PropsData GetPropsData(Props props)
         {
-            var builder = PropsData.CreateBuilder()
-                .SetClazz(props.Type.AssemblyQualifiedName)
-                .SetDeploy(GetDeployData(props.Deploy));
+            var propsData = new PropsData
+            {
+                Clazz = props.Type.AssemblyQualifiedName,
```

Comment:
@Horusiath @Aaronontheweb Should we use our own TypeQualifiedName instead of full name with the assembly version?